.circleci/config.yml

```diff
@@ -601,15 +605,17 @@ workflows:
     when: << pipeline.parameters.create-release >>
     jobs:
       - slack/approval-notification:
-          context: IEAT_keys
+          context:
+            - salesforce-ide
           message: Pending Approval for Creation of Release Branch
           channel: 'pdt_releases'
           color: '#0E1111'
       - hold: # Requires manual approval in Circle Ci
           type: approval
       - create-release-branch:
           e: default-executor
-          context: IEAT_keys
+          context:
+            - salesforce-ide
```

Comment:
I moved IEAT_keys -> salesforce-ide & added the NPM context to the publish jobs.  I think that covers it

Reply:
Before this name change, were Npm tokens defined in IEAT_keys?

Reply:
yup, now are found in NPM_Salesforce-Releases

Reply:
Maybe a silly question @gbockus-sf, but this is a new token, which is expected, right? Do we need to disable the old token once we retire IEAT_keys?

Reply:
The token is owned by the cli team and has already been rolled so the current value in IEAT_keys is already disabled/expired.  Good question though